cmd/thor/node/packer_loop.go

```diff
@@ -190,6 +191,10 @@ func (n *Node) pack(flow *packer.Flow) (err error) {
 		n.processFork(newBlock, oldBest.Header.ID())
 		commitElapsed := mclock.Now() - startTime - execElapsed
 
+		if newBlock.Header().Number() == n.forkConfig.GALACTICA {
```

Comment:
This means only the block proposer will see this, is this the designed feature?

Reply:
you're right, this is not what's expected. I'll move it into the node package

Reply:
Did you miss removing this snippet ?